Makefile

```diff
@@ -0,0 +1,34 @@
+review:
+	echo "setting Review $(env) environment"
+	$(eval env_config=review)
+	$(eval backend_key=-backend-config=key=$(env).terraform.tfstate)
+	$(eval export TF_VAR_paas_app_environment=$(env))
+
+qa:
+	$(eval env=qa)
+	$(eval env_config=qa)
```

Comment:
`env_config` was added to the tfvars, so there is no need to ad it here. `$(eval export TF_VAR_env_config=$(env_config))` can be moved to `review:`

Reply:
done please check....

Reply:
well it's still here: `$(eval env_config=qa)`

Reply:
and on line 39

Reply:
done.